book/src/drivers.md

```diff
@@ -81,6 +81,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Secrets Store CSI Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver) | `secrets-store.csi.k8s.io` | v0.0.10 | A Container Storage Interface (CSI) Driver for mounting secrets, keys, and certs stored in enterprise-grade external secrets stores as volumes. | Ephemeral | N/A | N/A | 
 [SmartX](http://www.smartx.com/?locale=en) | `csi-smtx-plugin` | v1.0 | A Container Storage Interface (CSI) Driver for SmartX ZBS Storage  | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion
 [StorageOS](https://docs.storageos.com/docs/platforms/kubernetes/install/) | `storageos` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for [StorageOS](https://storageos.com/) | Persistent | Read/Write Multiple Pods | Yes |
+[Storidge](https://docs.storidge.com/kubernetes_storage/install.html) | `csi.cio.storidge.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for [Storidge CIO](https://storidge.com/) | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion
```

Comment:
I'm slightly concerned that this linked documentation makes no mention of CSI Driver.

Reply:
The link to the csi driver install has been updated. Please check.

Reply:
I don't see any mentioning of CSI on this install page (https://docs.storidge.com/kubernetes_storage/install.html).
I think it may be better to replace it with this overview page (https://docs.storidge.com/kubernetes_storage/overview.html) which gives an overview of Storidge and its CSI integration.

Reply:
Thanks! Agree the overview page is a better link. It's been updated.

Reply:
I see it shows CSI driver install page now.  Looks good.